### PR TITLE
Tweak changelog

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/leveldown/issues/431

Direct link to this `CHANGELOG.md` https://github.com/Level/leveldown/blob/tweak-changelog/CHANGELOG.md

### CODE REVIEW

**Comment by ralphtheninja** [comment]
It's looking pretty neat now with links to users etc.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,598 +5,712 @@
...
-* `README` fixes (@ralphtheninja)
+
+-   Update dependencies ([**@ralphtheninja**](https://github.com/ralphtheninja))
+-   Use `prebuild --install` ([**@ralphtheninja**](https://github.com/ralphtheninja))
+-   `README` fixes ([**@ralphtheninja**](https://github.com/ralphtheninja))
 
 ### Fixed
-* Fix build problems on windows https://github.com/Level/leveldown/issues/247 (@obastemur)
+
+-   Fix build problems on windows [Level#247](https://github.com/Level/leveldown/issues/247) ([**@obastemur**](https://github.com/obastemur))
```
Comment (vweevers) [inline CHANGELOG.md]: Missing parentheses around issue

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,598 +5,712 @@
...
-* Add node 5 to prebuild config (abi 47) (@ralphtheninja)
+
+-   Bump dependencies ([**@ralphtheninja**](https://github.com/ralphtheninja))
+-   Bump `prebuild` for `--all` functionality ([**@ralphtheninja**](https://github.com/ralphtheninja))
+-   Add node 5 to prebuild config (abi 47) ([**@ralphtheninja**](https://github.com/ralphtheninja))
 
 ### Fixed
-* Fix build problems on windows https://github.com/Level/leveldown/issues/196 (@obastemur)
+
+-   Fix build problems on windows [Level#196](https://github.com/Level/leveldown/issues/196) ([**@obastemur**](https://github.com/obastemur))
```
Comment (vweevers) [inline CHANGELOG.md]: Missing parentheses around issue

**Comment by vweevers** [comment]
This is very nice indeed! I love remark :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -289,7 +289,7 @@
 
 ### Fixed
 
--   Fix build problems on windows [Level#247](https://github.com/Level/leveldown/issues/247) ([**@obastemur**](https://github.com/obastemur))
```
Comment (ralphtheninja) [inline CHANGELOG.md]: The reason I missed these is because they were plain urls and not in the `#NR` format.

**Pull request merged**